api/enki.py

```diff
@@ -626,12 +626,18 @@ def extract_circulation(self, primary_identifier, availability):
         if availability.get('accessType') == 'acs':
             drm_type = EnkiAPI.adobe_drm
         formats = []
-        formats.append(
-            FormatData(
-                content_type=Representation.EPUB_MEDIA_TYPE,
-                drm_scheme=drm_type
+        if formattype != {}:
+            if formattype == 'PDF':
+                content_type = Representation.PDF_MEDIA_TYPE
+            elif formattype == 'EPUB':
+                content_type=Representation.EPUB_MEDIA_TYPE
```

Comment:
This is basically fine, apart from checking `if formattype is None` instead of `if formattype != {}`, but if there's ever a new, unrecognized `formattype` this is going to blow up because `content_type` won't get defined.

Something like this would be more reliable:
```
content_type = None
if formattype == 'PDF':
    content_type = Representation.PDF_MEDIA_TYPE
elif formattype == 'EPUB':
    content_type = Representation.EPUB_MEDIA_TYPE

if content_type:
  ...
else:
   self.log.error("Unrecognized formattype: %s", formattype)
```

That way if there's an unexpected formattype, the title won't get a FormatData at all, and an error will be logged.